delft3dworker/views.py

```diff
@@ -74,16 +83,134 @@ class ScenarioViewSet(viewsets.ModelViewSet):
+                        logging.info(
+                            "Lookup value for parameter {}".format(key))
+
+                        # Find integers or floats
+                        try:
+                            value = float(value)
+                        except ValueError:
+                            pass  # no float? no problem
+
+                        # Create json lookup
+                        # q = {key: {'value': value}}
+
+                        # Not yet possible to do json queries directly
+                        # Requires JSONField from Postgresql 9.4 and Django 1.9
+                        # So we loop manually (bad performance!)
+                        wanted = []
+                        queryset = queryset.filter(parameters__icontains=key)
+                        for scenario in queryset:
+                            try:
+                                for pval in scenario.parameters[key]['values']:
+                                    if value == pval:
+                                        wanted.append(scenario.id)
+                            except ValueError:
+                                pass  # appearantly "values" does not exist, np
+
+                        queryset = queryset.filter(pk__in=wanted)
+
+                    # Key, min, max lookup
+                    elif len(p) == 3:
+
+                        key, minvalue, maxvalue = p
+                        logging.info(
+                            "Lookup value [{} - {}] for parameter {}".format(
+                                minvalue,
+                                maxvalue,
+                                key
+                            )
+                        )
+
+                        # Make into integers or floats
+                        # (no float values should throw exception)
+                        minvalue = float(minvalue)
+                        maxvalue = float(maxvalue)
+
+                        # Create json lookup
+                        # q = {key: {'value': value}}
+
+                        # Not yet possible to do json queries directly
+                        # Requires JSONField from Postgresql 9.4 and Django 1.9
+                        # So we loop manually (bad performance!)
+                        wanted = []
+                        queryset = queryset.filter(parameters__icontains=key)
+                        for scenario in queryset:
+                            try:
+                                for pval in scenario.parameters[key]['values']:
+                                    if minvalue <= pval <= maxvalue:
+                                        wanted.append(scenario.id)
+                            except ValueError:
+                                pass  # appearantly "values" does not exist, np
+
+                        queryset = queryset.filter(pk__in=wanted)
+
+            except:
+                logging.error("Something failed in search")
```

Comment:
`logging.exception` to keep the traceback, instead of just something failed. Errors should never pass silently?